intel-gpu-exporter.py

```diff
@@ -68,28 +117,54 @@ def update(data):
 
     igpu_rc6.set(data.get("rc6", {}).get("value", 0))
 
-if __name__ == '__main__':
-    logging.basicConfig(format='%(asctime)s - %(message)s', level=logging.INFO)
-    start_http_server(8080)
 
-    period = os.getenv("REFRESH_PERIOD_MS", 10000)
+def is_docker():
+    return os.path.exists("/.dockerenv")
```

Comment:
This should just be an env defined in the Dockerfile which you can check for in the script. No need to add or check for files.

Reply:
ohh, yeah that makes a lot more sense, i changed that.